spec/factories/user.rb

```diff
@@ -7,4 +7,13 @@
 
     sequence(:email) { |n| "user#{n}@example.com" }
   end
+
+  factory :mentee, class: "User" do
+    first_name { "Test" }
+    last_name { "user" }
+    role { :mentee }
+    password { "Testpassword1!" }
+    email {"mentee@example.com"}
```

Comment:
Have a look at how the sequence is used above. Because user emails must be unique if future tests require a second mentee to be the creation won't work as it won't pass the validation to be unique.